### DISCUSSION TITLE
QNX support for tokio

### DISCUSSION THREAD

**Discussion by AkhilTThomas** [tokio-rs/tokio#5565] [Ideas]
Since QNX support is available as of https://github.com/rust-lang/rust/pull/102701 it would be nice if tokio can also support nto target.

**Comment by Noah-Kennedy** [comment]
What would this entail? I'm not terribly familiar with this system.

**Reply by AkhilTThomas** [reply]
QNX is a big ecosystem in developing safety critical applications and this is a very popular library. Unfortunately it's a licensed software.

**Reply by Darksonn** [reply]
I have no idea what this is, but from your link, it sounds like only no_std is supported. Tokio cannot work without std.

**Reply by AkhilTThomas** [reply]
std support is there for the qnx neutrino 7.1 target.
QNX doc example: https://www.qnx.com/developers/docs/7.1/#com.qnx.doc.neutrino.lib_ref/topic/i/ionotify.html
